schemas/constructs/v1beta1/academy/openapi.yml

```
@@ -342,7 +342,7 @@ paths:
           content:
             application/json:
               schema:
-                type: object
+                $ref: '#/components/schemas/AcademyAdminSummary'

```

we can keep this as object for now , as it would break backend for now 
```suggestion
                type: object

```